### ISSUE TITLE
Rebrand to just "Did"

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#412]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
We should get rid of the 365 part and just call it _Did_.

### Description
There's no reason to call it Did 365. We should get rid of the 365 part and just call it _Did_. This requires to take over the domain of classic Did (http://did.puzzlepart.com/).

This requires the creation of a new web app in Azure and I think this should be running on Linux to avoid using `iisnode`.

### Tasks
- [x] Migrate TimeEntries from https://did.puzzlepart.com/ to azure table storage
- [x] Migrate TimeEntriesArchive from https://did.puzzlepart.com/ to azure table storage
- [x] Remove '365' part of logo
- [x] Remove '365' part of favicon
- [x] Remove '365' from app name
- [x] Rename GH-repo to `did`
- [x] Set new logo for app registration in Azure
